guides/common/modules/proc_cloning-ansible-roles-from-vcs.adoc

```diff
@@ -0,0 +1,24 @@
+[id="cloning-ansible-roles-from-vcs-{context}"]
+= Cloning Ansible roles from VCS
+
+You can clone Ansible roles to your {SmartProxy} from the {ProjectWebUI}.
+
+.Prerequisites
+* A {SmartProxy} with VSC integration for Ansible.
+For more information, see xref:setting-up-vsc-integration-for-ansible_{context}[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Configure* > *Ansible* > *Roles*
+. Click on `Clone from VCS...` in the toolbar.
+Follow the wizard to clone an Ansible role from VCS.
+
+Note: Currently, only links using *HTTPS* are supported
```

Comment:
```suggestion
Note that you can only clone Ansible roles through `http` and `https`.
```

I suppose this is because you cannot yet clone Ansible roles through SSH yet? @Thorben-D

Reply:
Correct, as of now, SSH is not supported.